identify-graphics-hittest/src/main/java/com/esri/arcgisruntime/sample/identifygraphicoverlay/MainActivity.java

```diff
@@ -70,45 +63,22 @@ protected void onCreate(Bundle savedInstanceState) {
         MapViewTouchListener mMapViewTouchListener = new MapViewTouchListener(this, mMapView);
         mMapView.setOnTouchListener(mMapViewTouchListener);
 
-        // work with the MapView after it has loaded
-        mMapView.addDrawStatusChangedListener(new DrawStatusChangedListener() {
-
-            @Override
-            public void drawStatusChanged(DrawStatusChangedEvent drawStatusChangedEvent) {
-                Log.i(TAG, "drawStatusChanged: " + drawStatusChangedEvent.getDrawStatus().toString());
-
-                if (drawStatusChangedEvent.getDrawStatus() == DrawStatus.COMPLETED) {
-                    // get center of MapView
-                    Polygon visibleArea = mMapView.getVisibleArea();
-                    Envelope polygonExtent = visibleArea.getExtent();
-                    Point center = polygonExtent.getCenter();
-                    // pass center of MapView to add graphic overlay
-                    addGraphicsOverlay(center);
-                }
-
-            }
-        });
+        addGraphicsOverlay();
     }
 
-    private void addGraphicsOverlay(Point center) {
-        // create values inside the visible area extent for creating graphic
-        double xValue = mMapView.getVisibleArea().getExtent().getWidth() / 5;
-        double yValue = mMapView.getVisibleArea().getExtent().getHeight() / 10;
-
-        // create point collection
-        PointCollection polyPoints = new PointCollection(mMapView.getSpatialReference());
-        polyPoints.add(new Point(center.getX() - xValue * 2, center.getY() - yValue * 2));
-        polyPoints.add(new Point(center.getX() - xValue * 2, center.getY() + yValue * 2));
-        polyPoints.add(new Point(center.getX() + xValue * 2, center.getY() + yValue * 2));
-        polyPoints.add(new Point(center.getX() + xValue * 2, center.getY() - yValue * 2));
-        // create polygon
-        Polygon polygon = new Polygon(polyPoints);
+    private void addGraphicsOverlay() {
+        // create the polygon
+        PolygonBuilder polygonGeometry = new PolygonBuilder(SpatialReferences.getWebMercator());
+        polygonGeometry.addPoint(-20e5, 20e5);
+        polygonGeometry.addPoint(20e5, 20.e5);
+        polygonGeometry.addPoint(20e5, -20e5);
+        polygonGeometry.addPoint(-20e5, -20e5);
 
         // create solid line symbol
-        int blue = Color.rgb(0, 0, 230);
-        SimpleLineSymbol solidLine = new SimpleLineSymbol(SimpleLineSymbol.Style.SOLID, blue, 4);
-        // create graphic from polygon and symbol
-        Graphic graphic = new Graphic(polygon, solidLine);
+        int yellow = Color.argb(4, 255, 255, 0);
```

Comment:
This is specifying an almost-completely-transparent fill symbol. Suggest Color.argb(255, 255, 255, 0) instead (or at least 128 for the alpha component).